release-notes/docs/3.10.0.md

```diff
@@ -0,0 +1,149 @@
+# 3.10.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Supporting remote first-time login](#supporting-remote-first-time-login)
+- [Supporting linked docs](#supporting-linked-docs)
+- [Privacy Policies](#privacy-policies)
+- [More granular replication depth configuration for offline users](#more-granular-replication-depth-configuration-for-offline-users)
+- [Improved geolocation data saved for reports](#improved-geolocation-data-saved-for-reports)
+- [Customizable header tabs icons](#customizable-header-tabs-icons)
+- [Improved performance of processing deleted docs](#improved-performance-of-processing-deleted-docs)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+Sentinel no longer advances its transition processing seq in `_local/sentinel-meta-data`. Instead, it advances the transition processing seq in `_local/transitions-seq`. Additionally, a new document `_local/background-seq` advances the deletion processing seq. 
+
+### Breaking changes
+
+#### Updates to meta databases replication
+
+The replication of users meta databases to the conglomerate, `medic-users-meta` database, is no longer configurable. 
+This task now runs every day, at 2am UTC, and replicates `feedback` and `telemetry` documents to `medic-users-meta` database.
+
+#### Android wrapper update required for new features
+
+Some features in 3.10 are only fully functional while using the latest release of medic-android.
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ ???? | 3.0+ |
+
+// todo medic-android version?? 
```

Comment:
It's backwards compatible, right? So I don't think we need to bump the minimum medic-android version. We just need to make it clear that certain features won't work without it.